sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj

```diff
@@ -21,7 +21,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="RL.Net" VersionOverride="0.1.0-preview3" />
+    <PackageReference Include="RL.Net" Version="0.2.0" />
```

Comment:
If you are doing this directly in the project it will need to be VersionOverride. I definitely think this is better then moving this preview dependency to the common props file but we still do need to get this dependency reviewed and approved and we should avoid shipping any stable versions of this package with this prerelease dependency.

Reply:
Thanks. Updated. Yes. Let's wait for the dependency review.

Reply:
@tg-msft -> Could you help check the comments to review this dependency?

Reply:
When we last discussed this in https://github.com/Azure/azure-sdk/issues/3877#issuecomment-1043638770, I believe the conclusion was you'd bring your dependency back to the Arch Board when ready because there were open questions like how to make this pay-for-play, what this looks like across our platform matrix, whether this works on other platforms, etc.  That still blocks your GA release per comments in https://apiview.dev/Assemblies/Review/35ddf25e338f4bb88d114ab43933c003.

That said, I have no concerns with folks updating the package in the repo to continue working on it though.  As @weshaggard suggested, this should definitely be scoped to just Personalizer.  (I'm not sure why anyone else would _want_ to take a dependency on it, but I'd rather we get a heads up if they tried.)